easybuild/easyconfigs/s/Seqtk/Seqtk-1.2-goolf-1.7.20-r94.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'Seqtk'
+version = '1.2'
+versionsuffix = '-r94'
```

Comment:
why this versionsuffix? what does it refer to?

there's a proper version on GitHub, and you're not using `r94` or `94` anywhere?